crates/miden-objects/src/block/proven_block.rs

```diff
@@ -50,6 +50,9 @@ pub struct ProvenBlock {
     /// The aggregated and flattened transaction headers of all batches in the order in which they
     /// appeared in the proposed block.
     transactions: OrderedTransactionHeaders,
+
+    /// The commitment to the proof of the block.
+    proof_commitment: Word,
```

Comment:
I don't think we need this here. Eventually, we'll have to add a `proof` field here - but that would be a full proof, rather than just a commitment.